tests/b2share_unit_tests/upgrade/test_upgrade.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of EUDAT B2Share.
+# Copyright (C) 2017 CERN.
+#
+# B2Share is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# B2Share is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with B2Share; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Test db alembic functionality."""
+from flask_cli import ScriptInfo
+from click.testing import CliRunner
+from invenio_db import InvenioDB, db
+from b2share.modules.upgrade.cli import run
+from b2share_unit_tests.helpers import db_create_v2_0_1, \
+    remove_alembic_version_table
+from sqlalchemy.engine.reflection import Inspector
+import pytest
+
+
+def test_alembic(app):
+    """Test alembic recipes."""
+    ext = InvenioDB(app, entry_point_group=False, db=db)
+
+    with app.app_context():
+        if db.engine.name == 'sqlite':
+            raise pytest.skip('Upgrades are not supported on SQLite.')
+        db.drop_all()
+        remove_alembic_version_table()
+        ext.alembic.upgrade()
+        # downgrade to root revision which is in invenio-db
+        ext.alembic.downgrade(target='96e796392533')
+
+        insp = Inspector.from_engine(db.engine)
+        remaining_table = insp.get_table_names()
+        assert remaining_table == ['alembic_version']
+        remove_alembic_version_table()
+        insp = Inspector.from_engine(db.engine)
+        assert not insp.get_table_names()
+
+
+def test_upgrade_from_early_version(app):
+    """Test upgrading B2Share from version 2.0.1."""
+    ext = InvenioDB(app, entry_point_group=False, db=db)
+
+    with app.app_context():
+        if db.engine.name == 'sqlite':
+            raise pytest.skip('Upgrades are not supported on SQLite.')
+        # Bring DB to v2.0.1 state
+        db.drop_all()
+        db_create_v2_0_1()
+
+        # The b2share_migrations table is missing in v2.0.1
+        # and will show up as a suggested revision
```

Comment:
Why does it suggest only "b2share_migrations"? Isn't the alembic_version table missing at that point? Every alembic script should be suggested.